assets/css/pages-frontend.scss

```diff
@@ -1,3 +1,5 @@
+@import 'fontawesome';
```

Comment:
The legacy styles aren't loaded on the new Learner Courses page.

It introduces the `fontawesome` here to be used in this page (Sensei WCPC will introduce a message of expiration courses with an icon).